vroom/main.vroom

```diff
@@ -185,8 +220,8 @@ them are available, the message will show a line for each.
   :let fake_format2 = copy(fake_format)
   :let fake_format2.name = 'fake-format2'
   :let fake_format2.setup_instructions = 'LMGTFY'
-  :call codefmtlib#AddFormatter(fake_format2)
+  :call maktaba#extension#GetRegistry('codefmt').AddExtension(fake_format2)
 
   :FormatCode
-  ~ Formatter "fake-format" is not available. Setup instructions: RTFM
   ~ Formatter "fake-format2" is not available. Setup instructions: LMGTFY
+  ~ Formatter "fake-format" is not available. Setup instructions: RTFM
```

Comment:
What's up with the order here? Not that it's a huge deal, but I thought we preserved order for first-wins so I'm surprised it changed here.